src/apps/LandingRegister/Header.js

```diff
@@ -4,55 +4,88 @@ import {IconButton} from 'material-ui';
 
 export default Radium(() => {
   const styles = {
-    headerDiv: {
+    header: {
+      marginTop: 21,
+      marginBottom: 71,
+      width: '100%',
+      display: 'flex',
       justifyContent: 'flex-end',
-      display: 'flex'
+      '@media (max-width: 750px)': {
+        justifyContent: 'space-between',
+        marginBottom: 40
+      }
+    },
+    headerLogo: {
+      display: 'none',
+      '@media (max-width: 750px)': {
+        display: 'flex',
+        paddingLeft: 16
+      }
+    },
+    headerLogoImg: {
+      width: 175,
+      height: 37,
+      '@media (max-width: 330px)': {
+        width: 130,
+        height: 27
+      }
     },
     iconStyle: {
-      fontSize: 38.5,
-      color: '#454545',
-      margin: '-4px 0px 0px -7px'
+      fontSize: 30,
+      color: '#454545'
     },
     buttonStyle: {
-      margin: '28px 7px 28px 0px',
-      padding: 0
+      padding: '0px 15px 0px 0px',
+      width: 'auto',
+      height: 'auto',
+      margin: 0
+    },
+    iconImg: {
```

Comment:
you shouldn't modify `IconButton` by setting `width` and `height`, check how the button looks like after clicking on it